### PR TITLE
Tool Damage Buff

### PR DESCRIPTION
**Pull request by Geevies**

### CODE REVIEW

**Comment by Geevies** [comment]
!review

**Comment by alsoandanswer** [comment]
I think there definitely should be more thought put into this.

I mean, the reason why they're pretty stupid weak in the first place is cause they're extremely easy and obtain, so some greytider can't rip you limb from limb with a crowbar and a cable cuff.

**Comment by Geevies** [comment]
Three extra damage won't cause that much more of a problem, since things that can do that and more are also easily attainable, such as by breaking windows, stealing the hatchets from the garden, grabbing a fire extinguisher, etc.

The only argument I can see that's valid here would be the impact wrench which is at 10 force. I suppose this can be balanced by giving them a power cell a-la stun batons. Thoughts?

File: code/game/objects/items/weapons/tools.dm [hunk 1]
```diff
@@ -129,13 +129,13 @@
 		item_state = "cutters_yellow"
 	..()
 
-/obj/item/wirecutters/attack(mob/living/carbon/C as mob, mob/user as mob, var/target_zone)
+/obj/item/wirecutters/attack(mob/living/carbon/C, mob/user, var/target_zone)
 	if(user.a_intent == I_HELP && (C.handcuffed) && (istype(C.handcuffed, /obj/item/handcuffs/cable)))
-		usr.visible_message("\The [usr] cuts \the [C]'s restraints with \the [src]!",\
-		"You cut \the [C]'s restraints with \the [src]!",\
-		"You hear cable being cut.")
+		user.visible_message(SPAN_NOTICE("\The [usr] cuts \the [C]'s restraints with \the [src]!"),\
```
Comment (NonQueueingMatt) [inline code/game/objects/items/weapons/tools.dm]:
```suggestion
		user.visible_message(SPAN_NOTICE("\The [user] cuts \the [C]'s restraints with \the [src]!"),\
```

File: code/game/objects/items/weapons/tools.dm [hunk 2]
```diff
@@ -641,8 +639,9 @@
 	icon = 'icons/obj/tools.dmi'
 	icon_state = "powerdrillyellow"
 	var/drillcolor = null
-	force = 3
-	w_class = 2
+	force = 10
```
Comment (NonQueueingMatt) [inline code/game/objects/items/weapons/tools.dm]: I think this should be on the same level as the others, so like 8 force. It's not exactly physically different or stronger.

Comment (Geevies) [inline code/game/objects/items/weapons/tools.dm]:
```suggestion
	force = 8
```

**Pull request merged**